Jenkinsfile-win64

```
@@ -138,7 +138,7 @@ def TestWin64GPU(args) {
     """
     echo "Installing Python dependencies..."
     bat """
-     conda activate && conda install -y hypothesis && conda upgrade scikit-learn pandas numpy hypothesis
+     conda activate && conda install -y hypothesis cupy && conda upgrade scikit-learn pandas numpy hypothesis cupy
```

cuPy version on Conda is quite old. I suggest install cuPy from pip
```suggestion
     conda activate && conda install -y hypothesis && conda upgrade scikit-learn pandas numpy hypothesis  && python -m pip install cupy-${cuda_short_ver}
```